packages/core-utils/src/time.js

```diff
@@ -92,8 +97,8 @@ export function formatDuration(seconds) {
  * @param {number} seconds duration in seconds
  * @returns {string} formatted text representation
```

Comment:
Add new param to JSdoc.

Reply:
Added in 607cb0f2263907b88396ccd78574129c94d0b1a2